src/main/cc/maze/cell/cell.h

```diff
@@ -2,8 +2,29 @@
 #define CC_CELL_CELL_H_
 
 // Abstract class to represent a cell.
-public class Cell {
-	 	
+class Cell {
+	// Creates a Cell object at (x,y)
+	Cell(int x, int y);
+
+	// Gets the x offsest of a Cell
+	int GetLocationX();
+
+	// Gets the y offset of a Cell
+	int GetLocationY();
+
+	// Gets the status of whether a Cell has been visited
+	bool IsVisited();
+
+	// Sets a Cell to visited (true)
```

Comment:
So I think in this comment and in the one below it we would just want to say that it visits a cell. That way you arent bound to setting the cell to visited (true). Because how do we know that is what we might want to do in the future when we call this function. In my mind to people using the cell when they call visit they expect it to visit the cell, but they have no notion of what that will actually mean. By saying it sets a variable to true you are like saying that this class has a private member variable called visited.